### ISSUE TITLE
Accessibility: Setup UI Bug 

### ISSUE THREAD

**Issue by jhkimnew** [microsoft/IIS.Administration#221]
Repro Steps:

Hit the URL "https://manage.iis.net" 
Tab till "Download Microsoft IIS Administration" button and press enter and run "IISAdministrationSetup.exe" file
Turn on high contrast black mode
Verify in high contrast black mode, user is able to see "I agree to the license terms and condition" beside the checkbox or not

Expected Result:
In high contrast black mode, user should able to see "I agree to the license terms and condition" beside the checkbox

Actual Result:
In high contrast black mode, unable to see "I agree to the license terms and condition" beside the checkbox

![image](https://user-images.githubusercontent.com/20650163/49771616-ca11da00-fc9e-11e8-9a3a-2030f0ae421a.png)

**Comment by jhkimnew** [comment]
Fix below issue as well.

https://github.com/Microsoft/IIS.WebManager/issues/214

**Comment by yaqiyang** [comment]
Closed. New features are not going to be implemented in the near future
